src/Lib/Autofac.Extras.Quartz/Autofac.Extras.Quartz.csproj

```diff
@@ -19,10 +20,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac" Version="[7.0.0, 8.0.0)" />
-    <PackageReference Include="JetBrains.Annotations" Version="2022.3.1" />
-    <PackageReference Include="Quartz" Version="[3.6.2, 4.0.0)" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1">
+    <PackageReference Include="Autofac" Version="[7.0.0, 8.0.0]" />
```

Comment:
Maybe, it's better to specify only minimum version: [7.0.0,)? Or exclude next major Autofac update: [7.0.0, 9.0.0)? I suppose, upcoming minor and patch updates shouldn't break anything.
The same thoughts about Quartz dependency.

Reply:
Thank you for contribution, I'm upgrade dependencies and update .net frameworks this week

Reply:
@jafin @Alexander-Leontiev 
I published new pre-release version. Since it incorporates new major version of the Autofac, I bumped major version of the package, please git [10.0.0-alpha0006](https://www.nuget.org/packages/Autofac.Extras.Quartz/10.0.0-alpha0006) a try

.NET 7 has not reached it's EOL, I would keep it for at least 3 month past EOL.

Reply:
@shatl thanks! What the plans for stable v10 release?